NOTICE

```diff
@@ -3,22 +3,3 @@ Copyright 2024-2025 The Apache Software Foundation
 
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
```

Comment:
This is correct. The `NOTICE` file should only contains the `NOTICE` of Apache licensed projects where the code has been copied (`NOTICE` should not be used for other license types), if provided, and only the relevant part of our project.
In the case of ASF projects (like Apache Arrow), it's not needed to copy `NOTICE` file as it's already covered by `This product includes software developed at The Apache Software Foundation (http://www.apache.org/)`.